### DISCUSSION TITLE
Serverless and collections

### DISCUSSION THREAD

**Discussion by cfjedimaster** [11ty/eleventy#2247] [Q&A]
It appears as if serverless templates do not have access to collections. Is that expected, and if so, can we quickly add it to the documentation?

**Comment by raffaellarinaldi** [comment]
Hello, have you found a solution for this?
I am having the same problem. I need to show collection content in a serverless page and I can't get it to work.
I'm not sure if it's expected behaviour or not, if not I'll open my own thread and post my code. 
Thanks!

**Comment by cfjedimaster** [comment]
No - let me go ahead and file an issue for this. If it's expected, then it should be documented.

**Reply by cfjedimaster** [reply]
Reference: https://github.com/11ty/eleventy/issues/2268

**Reply by cfjedimaster** [reply]
FYI, I think Zach may have referenced this here: https://twitter.com/eleven_ty/status/1459195323545526275. If I read this right, the solution is: Use a template to output collections in JSON format. Use the copy option of the plugin to copy that thing. And then maybe you use a filter/shortcode that reads that in and is available in your template? The last part doesn't quite make sense. I've pinged for more info and will report back.

**Reply by cfjedimaster** [reply]
Ok, hashed out more with Zach on twitter - and if you look at the linked bug above - it's going to be addressed. There IS a workaround, but I'd probably suggest waiting (so does Zach :).

**Reply by raffaellarinaldi** [reply]
Thanks! I'll wait for this to be addressed. I'll use github actions in the meantime :)
